congressional-support.yaml

```diff
@@ -0,0 +1,119 @@
+- type: house
+  name: House Documents Repository
+  url: http://docs.house.gov/
+
+- type: house
+  name: Office of Compliance
+  url: http://www.compliance.gov/
+
+- type: house
+  name: Clerk of the House
+  url: http://clerk.house.gov/
+
+- type: house
+  name: Chief Administrative Officer
+  url: http://cao.house.gov/
+
+- type: house
+  name: Sergeant at Arms
+  url:
+
+- type: house
+  name: Chaplain
+  url: http://chaplain.house.gov/
+
+- type: house
+  name: Inspector General
+  url: http://www.house.gov/content/learn/officers_and_organizations/inspector_general.php/
```

Comment:
I don't know you'd have ended up with a trailing slash on this URL in your browser. Mind axing at least this one before merge?